index.js

```diff
@@ -0,0 +1,145 @@
+const baseUrl = 'https://trektravel.herokuapp.com/trips'
+
+// Get the modal
+const modal = document.getElementById('myModal');
+
+// Get the button that opens the modal
+const btn = document.getElementById("myBtn");
+
+// Get the <span> element that closes the modal
+const span = document.getElementsByClassName("close")[0];
+
+
+const successCallback = function(response){
+  console.log("POST request to reserve a spot on a trip was successful");
+  console.log(response);
+  let name = $('#trip-form').data('name');
+  $('#message').html(`Congratulations! You successfully reserved a spot on ${name}!`);
+
+  // When the user clicks the button, open the modal
+  modal.style.display = "block";
+};
+
+const continents = {
+  'Africa': 'africa.png',
+  'Antarctica': 'antarctica.png',
+  'Asia': 'asia.png',
+  'Australasia': 'australia.png',
+  'Europe': 'europe.png',
+  'North America': 'north_america.png',
+  'South America': 'south_america.png'
+};
+
+const continentImgInfo = function(trip){
+  if (continents[trip]){
+    return `<img src="img/${continents[trip]}" alt="${trip} picture" class="continent"/>`;
+  }
+};
+
+$(document).ready(()=> {
+  $('#reserve-form').hide();
+  $('#content').hide();
+  $('#show-trip').hide();
```

Comment:
To avoid a flash of these when the page first loads, instead of calling `.hide()` here in `$(document).ready` you should add a CSS rule `display: none` (this is what `.hide()` does behind the scenes).